src/python/tests/core/platforms/android/adb_test.py

```diff
@@ -246,3 +248,15 @@ def test_apk_path_in_arg(self):
     """Test apk path passed as argument."""
     self.assertEqual(
         adb.get_package_name(self.test_apk_path), self.test_apk_pkg_name)
+
+
+@test_utils.slow
```

Comment:
this slow can be removed after mocking time.sleep.

Reply:
Will remove this depending on what we decide to do here.